internal/third_party/dep/fs/fs.go

```diff
@@ -164,7 +164,8 @@ func copyFile(src, dst string) (err error) {
 				//
 				// ERROR_PRIVILEGE_NOT_HELD is 1314 (0x522):
 				// https://msdn.microsoft.com/en-us/library/windows/desktop/ms681385(v=vs.85).aspx
-				if lerr, ok := err.(*os.LinkError); ok && lerr.Err != syscall.Errno(1314) {
+				var lerr *os.LinkError
+				if errors.As(err, &lerr) && !errors.Is(lerr.Err, syscall.Errno(1314)) {
```

Comment:
```suggestion
				if lerr := os.LinkError{}; errors.As(err, &lerr) && !errors.Is(lerr.Err, syscall.Errno(1314)) {
```

Perhaps (not even sure if it is worth it), the "hanging" `var lerr` could be included in the conditional?